style_guide.md

```diff
@@ -1,26 +1,100 @@
+- be written in a neutral voice, not for a specific audience. E.g for 'foundingDate' - 'When the entity was founded, in ISO 8601 format' not 'When the entity was founded, please provide this in ISO 8601 format'
+
+Subsequent sentences may provide information or guidance to assist publishers to use the field effectively or users to interpret the field effectively. Guidance sentences should be grounded in clear user needs and implementation experience of common pitfalls or errors.
+
+Descriptions with a link to a codelist should be phrased as - '\<description\>, using the \<name\> codelist.'
+
+Descriptions should also:
+- refer to property names using backquotes E.g. "\`id\` is required" (*wip - why backquotes specifically?*) 
+- refer to property values, whether example free text strings or values from a codelist, using single quotes. E.g. "the given name for Johann Sebastian Bach is 'Johann Sebastian'"
+- use plain text only, without markup (eg. no \[Markdown\](https://en.wikipedia.org/wiki/Markdown) or similar syntax)
+- place URLs in brackets by the appropriate word or phrase or at the end of the sentence
+
+Descriptions should not:
+- include duplications of the codelist as part of the description
+- link to external websites
+- only restate the title or name E.g 'Address type' - 'The address type'
+- explicity state whether a field is required or optional (this is indicated by an asterisk)
+- declare the type of the field (this is indicated above the description) 
+
+Assuming the rest of the guidance is followed, it is recommended to start the description with:
+- “Whether”, for a boolean field.
+- “The” with a plural noun phrase, for the description of an array of strings.
+
+  
+### 
 
-This style guide is a draft to be discussed and not yet implemented across the BODS schema and documentation (as of May 2019, BODS v0.2). It can be used to guide editing and content writing. The Guide is divided into separate, though related, sections: one for the BODS schema and one for the BODS documentation.
-
-## BODS Schema
+**WIP notes**
 
-- Object and property names, and definition entry names to be camelCase, e.g. interestedParty
+*Adapt and adopt [OCDS style guidance for property descriptions](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#field-and-code-descriptions) ?*
 
-- Object and property titles: first word to be upper case. E.g. 'Interest' or 'Ownership-or-control statement'.
+'The top level of the schema is split between properties and definitions. The latter contains objects that may be re-used, by reference, in multiple locations across the schema. Each of these can be thought of as a “Class”, and its name is capitalized accordingly. Whenever you consider that an object or structure might be re-used in a different area of the standard, it should be included in definitions.'
```

Comment:
This is a point about schema design... so arguably should go elsewhere in the Handbook.